src/components/pages/AdminAddCourse/AdminAddCoursesForm.js

```diff
@@ -159,6 +159,23 @@ function AdminAddCoursesForm(props) {
                   style={{ height: 100, fontSize: '1.1rem' }}
                 />
               </Form.Item>
+              <Form.Item
+                label={
+                  <label style={{ color: '#096A70', fontSize: '1.1rem' }}>
```

Comment:
I think styles that are being applied repeatedly through the file would be better placed in a .css or .less file to keep it DRY but also to make it easier to change in the future. I'm always a proponent of keeping styling in one place so that if there's ever any issue with it, it's easier to hunt down the problem